contracts/src/chainlink/cairo/ocr2/aggregator.cairo

```diff
@@ -1109,10 +1079,18 @@ end
 
 # --- Transmitter Payment
 
-func calculate_reimbursement() -> (amount : felt):
-    # TODO:
-    let amount = 0
-    return (amount)
+const MARGIN = 115
+
+func calculate_reimbursement{range_check_ptr}(
```

Comment:
We should consider making some of these coefficients configurable via `set_billing`